### DISCUSSION TITLE
X Buy/Sell To Store

### DISCUSSION THREAD

**Discussion by kfc69bj** [runelite/runelite#18429] [Ideas]
Can We Get A Feature That Was In RuneScape Before EOC That Allowed Us To Sell To Stores Easier? We Had A Feature And Now We Don't And Would Like To See It Return. It Would Help People Out So Much.

**Comment by raiyni** [comment]
I don't know what you are requesting but you can menu swap the buy/sell menus in stores.

**Reply by kfc69bj** [reply]
Thank Just Saw It As You Said. Big Thanks
